docker-for-windows/index.md

```diff
@@ -707,6 +707,63 @@ Check out these [Docker Cloud topics](/docker-cloud/index.md) to learn more:
 Need a direct link to Cloud? [Take me to Docker
 Cloud](https://cloud.docker.com/){: target="_blank" class="_" }.
 
+### Kubernetes
+
+**Kubernetes is only available in Docker for Windows 18.02 CE Edge users who
+participants in the [Docker Beta program](https://beta.docker.com/). To access
+beta builds, you must be signed in with your Docker ID within Docker for Windows.
+Select ![whale menu](/docker-for-windows/images/whale-x.png){: .inline} -> Sign in / Create Docker ID
+from the menu bar.**
+
+The addition of Kubernetes support increases the total binary size by about
```

Comment:
Same as for mac : 300 MB is the size of the download when enabling Kubernetes ; maybe to mention below next to "an internet connection is required"